tests/main.c

```diff
@@ -2,7 +2,8 @@
 
 #include <getopt.h>
 
-TEST(Aarch64MinGW, BigDataTest);
```

Comment:
`Aarch64` should not be present in the test name. Changes in this file are just a sneak peak how it should be re-factored. I can either revert, if declined, or move to a separate PR, if accepted.